.zuul.yaml

```diff
@@ -8,15 +78,21 @@
         - flake8
         - hadolint
         - yamllint
+        - container-image-ceph-ansible-build-pacific
+        - container-image-ceph-ansible-build-quincy
     gate:
       jobs:
         - ansible-lint
         - flake8
         - hadolint
         - yamllint
+        - container-image-ceph-ansible-push-pacific
```

Comment:
We don't actually use the gate pipeline as of now. Also I think access to secrets needs to happen in post, so move these jobs there.

Reply:
Ok

Reply:
Now you dropped the jobs instead of creating a post section, do you really want to rely only on daily builds?